src/js/lib.js

```diff
@@ -617,6 +617,18 @@ vjs.round = function(num, dec) {
  * @private
  */
 vjs.createTimeRange = function(start, end){
+  if (start === undefined && end === undefined) {
```

Comment:
Is there a reason why we wouldn't throw an error when you try to create a time range with no start/end values, instead of creating one that throws errors later?